### ISSUE TITLE
Hey this looks pretty cool 

### ISSUE THREAD

**Issue by Huxpro** [lynn9388/huxpro#1]
Hey thanks for making this. I've never heard of [remote theme](https://github.blog/2017-11-29-use-any-theme-with-github-pages/) before but it sounds really promising. Would you be interested in some collaboration and make this an official and recommended way to adopt the blog theme? Feel free to friend me on WeChat (my username is the name of this repo XD) and we can discuss more details later.

**Comment by lynn9388** [comment]
Of course, I have added you in WeChat, please check the application message.
